docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Control CPU Management Policies on the Node
+---
+
+* TOC
+{:toc}
+
+Kubernetes keeps many aspects of how the pod executes on the node abstracted
+away from the user. This is by design.  However, some workloads require
+stronger guarantees in terms of latency and/or performance in order to operate
+acceptably.  The kubelet provides methods for enabling more complex workload
+placement policies while keeping the abstraction free from explicit placement
+directives.
+
+## CPU Management Policies
+
+By default, the kubelet uses [CFS quota](https://en.wikipedia.org/wiki/Completely_Fair_Scheduler)
+to enforce pod CPU limits.  When the node is running many CPU bound pods, this
```

Comment:
When a node runs many CPU bound pods, the workload can be moved to different CPU cores depending on whether a pod is throttled and which CPU cores are available at scheduling time.